crates/iracing-data-api-client/README.md

```
@@ -0,0 +1,216 @@
+This API client was generated by the [OpenAPI Generator](https://openapi-generator.tech) project.  By using the [openapi-spec](https://openapis.org) from a remote server, you can easily generate an API client.
+
+- API version: 0.0.1
+- Package version: 0.0.1
+- Generator version: 7.17.0
+- Build package: `org.openapitools.codegen.languages.RustClientCodegen`
+
+## Installation
+
+Put the package under your project folder in a directory named `iracing-data-api-client` and add the following to `Cargo.toml` under `[dependencies]`:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)